gap/base/recognition.gd

```diff
@@ -155,17 +155,17 @@ DeclareAttribute( "ImageRecogNode", IsRecogNode, "mutable" );
 ## <#/GAPDoc>
 DeclareAttribute( "KernelRecogNode", IsRecogNode, "mutable" );
 
-## <#GAPDoc Label="RIParent">
+## <#GAPDoc Label="ParentRecogNode">
```

Comment:
These renames potentially break existing code using recog.

Is there a strong reason why we don't use `DeclareObsoleteSynonym` resp. `DeclareObsoleteSynonymAttr` for these (and other, previously renamed things)?

Reply:
I added synonyms for the renames in this PR and am currently opening a PR for the renames that are already in master.